server/helpers/auth/strategies/googleStrategy.js

```diff
@@ -0,0 +1,112 @@
+import passport from 'passport';
+import googleStrategyOauth2 from 'passport-google-oauth';
+import bcrypt from 'bcryptjs';
+import models from '../../../models';
+
+/**
+ * @description A class that implements google strategy for passport
+ */
+export default class GoogleLogin{
+
+  /**
+   * @description Generate a random password for every user authenticated
+   * @returns {function} hashPassword
+   */
+  static generatePassword() {
+    const alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+    const numbersSpecialChars = '0123456789*&^%$##@!';
+    const random = alphabet + numbersSpecialChars;
+    let password = '';
+    for (let i = 0; i < 8; i++) {
+      password += random.charAt(Math.floor(Math.random() * random.length));
+    }
+    return this.hashPassword(password);
+  }
+
+  /**
+   * @description Generate a random password for every user authenticated
+   * @param {string} password
+   * @returns {function} hashPassword
+   */
+  static hashPassword(password) {
+    const salt = bcrypt.genSaltSync(10);
+    return bcrypt.hashSync(password, salt);
+  }
+
```

Comment:
Good one @JohnMadakin. There is a helper function in the codebase that can do this. `server/helpers/passwordHash`

Reply:
done. thanks for your observation